docs/usage/security-modes.md

```diff
@@ -0,0 +1,52 @@
+# Security modes
+
+This section describes the different **Security mode** options for connectivity that can be selected for Liqo, regulating connectivity restriction for intra-cluster traffic.
+
+## Overview
+
+Actually, Liqo provides two security modes:
+
+* **Full pod-to-pod connectivity**: represents the native behaviour of Liqo, there are no restriction in terms of connectivity, and all pods of the first cluster can connect to all pods of the second cluster (and vice versa).
+* **Intra-cluster traffic segregation**: full pod-to-pod connectivity is enabled only within real clusters; regarding intra-cluster traffic, local cluster has the possibility to contact only its offloaded pods, while remote cluster is allowed to contact exclusively services offloaded on it
+
+## Selection of the Security mode
+
+The desired Security mode can be selected with a **feature flag** at install time (see the [reference](/installation/install.md)):
+
+* `--set networking.securityMode=FullPodToPod` to select **full pod-to-pod connectivity (default mode)**
+* `--set networking.securityMode=IntraClusterTrafficSegregation` to select **intra-cluster traffic segregation**
+
+```{admonition} Note
+Currently, the selection of security mode is possible only at installation time. In future, this feature will be extended to be configured at run-time, e.g., each time you setup a new peering.
+```
+
+## Connectivity matrix
```

Comment:
When introducing the pictures, we have to define clearly all the components in there.
E.g., what are "clusters", what is a "virtual cluster", what is "I1" and "I2", and so on. We cannot give for granted that readers know what we have in mind.